package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "photo-gallery-service",
+  "version": "1.0.0",
+  "description": "Photo gallery display and modal from guest experience photos",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "jest --coverage",
+    "build": "webpack -d --mode development --watch --config webpack.config.js",
+    "start": "nodemon ./server/index.js",
+    "seed": "node ./database/seed.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/HRR43-Team-Leia/Photo-Gallery-Service.git"
+  },
+  "author": "Katie Swanson",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/HRR43-Team-Leia/Photo-Gallery-Service/issues"
+  },
+  "homepage": "https://github.com/HRR43-Team-Leia/Photo-Gallery-Service#readme",
+  "devDependencies": {
+    "@babel/core": "^7.8.3",
+    "@babel/preset-env": "^7.8.3",
+    "@babel/preset-react": "^7.8.3",
+    "babel-loader": "^8.0.6",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
+    "css-loader": "^3.4.2",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
+    "jest": "^25.1.0",
+    "nodemon": "^2.0.2",
+    "webpack": "^4.41.5",
+    "webpack-cli": "^3.3.10"
+  },
+  "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "axios": "^0.19.2",
+    "babel-jest": "^25.1.0",
+    "express": "^4.17.1",
+    "mysql": "^2.18.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-test-renderer": "^16.12.0"
```

Comment:
should this be in devDependencies?

Reply:
Probably :) I'll move them all over, good catch!